iron-request.html

```diff
@@ -151,7 +151,7 @@
       var xhr = this.xhr;
 
       if (xhr.readyState > 0) {
-        return;
```

Comment:
Just because of the return type of the function, which translates to "Promise or null". We could also have changed the return type to {Promise|undefined} instead.